ament_index_cpp/include/ament_index_cpp/get_resource.hpp

```diff
@@ -15,10 +15,10 @@
 #ifndef __ament_index_cpp__get_resource__h__
 #define __ament_index_cpp__get_resource__h__
 
-#include <string>
-
 #include <ament_index_cpp/visibility_control.h>
 
+#include <string>
+
```

Comment:
I have updated the includes of non-system headers to use quotes instead of angled brackets. This makes cpplint happy.